src/index_result.c

```diff
@@ -7,6 +7,7 @@
  * GNU Affero General Public License v3 (AGPLv3).
 */
 #include "index_result.h"
+#include "redisearch_rs/headers/types_rs.h"
```

Comment:
```suggestion
#include "types_rs.h"
```
This should also work